### ISSUE TITLE
Add coolapk.com into "Search in Stores" menu

### ISSUE THREAD

**Issue by sandriaas** [allangoing/apkshellext#98]
can we add coolapk.com into "Search in Stores" menu? the URL would be like this https://www.coolapk.com/apk/{package_name}

**Comment by blackcrack** [comment]
naa, it's like a Latain-language domain for a couples.. 
the internet language is English, i be a german so what.. i also not for a german domain of apk's
it is a matter of courtesy to have a common language, but this is not Chines or Japanese..
English is the first language of the Internet and the first becomes the surcharge..
so..

best

**Comment by sandriaas** [comment]
i meant, there are apps you can't find on playstore but it's available on coolapk, like many of xposed modules app. or maybe just simply add google search into the menu

**Comment by blackcrack** [comment]
what interesting this \*beep* Chinese page, and you make advertising with this \*beep*

maybe is this a Chinese warez page and this is not supporting, i can it not read and i want it not, please close this "issue" 

edit: huh.. i should drink less beer..


@kkguo

**Comment by sylveon** [comment]
What the hell is this gibberish

**Comment by Masamune3210** [comment]
> what interesting this *beep* Chinese page, and you make advertising with this *beep*
> 
> maybe is this a Chinese warez page and this is not supporting, i can it not read and i want it not, please close this "issue" or i tell FBI, CIA and maybe homlelandsecurity interferes too and the owner Microsoft of this Github Servers could i tell you be a troublemaker with a Chinese warez page and report it .. then could be block or delete your account and you be wanted because you be a pirate and a prisoner.. Because you have a IP and be logged by every post with time, what do you think know the Servers where have to go this page informations to send to your browser, yes, with a ip and the ip is logged by time too.. so, it is know from where you posting.
> and.. the name Mitnick is a well known special.. from the past....
> the 2nd.. the Chinese Facilitys could it found also interesting what do you here ..
> 你看到範圍了嗎？
> so, decide you..
> 
> @kkguo

Literally not one thing you said even slightly makes sense, please quit spamming the issue with nonsense. Especially if it's xenophobic nonsense!

**Comment by blackcrack** [comment]
@Masamune3210 , do you drink Alcohol ?
by the way, if adverting supported, then F-Droid, APKpure would have been in it for a long time.
best

**Comment by Masamune3210** [comment]
No, but I'm 90% sure you shouldn't either from the comment edit....
Pro Tip: Don't Drink and Internet

**Comment by blackcrack** [comment]
@Masamune3210 .. 
Pro Tip: ... damn.. and i already  don't drive a car any more with alcohol...
\*g*
best

**Comment by kkguo** [comment]
@blackcrack Hey bro, don't be personal and don't be rude. @andrizmitnick I will make the search menu customizable, so you can search with whatever you want.

**Comment by blackcrack** [comment]
@kkguo  good to see you in Corona times Chief :smiley: 
... everything good with you  ?

**Comment by kkguo** [comment]
@blackcrack yup, everything fine, except the time went too fast..

**Comment by blackcrack** [comment]
> 
> 
> @blackcrack yup, everything fine, except the time went too fast..  

@kkguo 
up to yesterday one week in the hospital, my Bodyengine(Heart) has only run up to 37% of 100%,
now all fine again.. (corona=> a weak heart ignition =>then i have take a strong penicillin from the Dr. .. because the felt pneumonia and get water in the lung.. i could sleep right or left side, then i have call the ambulance over the german 116117 sos call and they brought me to the Hospital.. ..
so, stay healthy and take FFP2 Mask's !! ;)
best

**Comment by kkguo** [comment]
Pray for you my bro.
You are the second one I heard who personally experienced recently.
Take care everyone.
